pkg/identityserver/end_device_registry.go

```diff
@@ -181,7 +182,9 @@ func (is *IdentityServer) setFullEndDevicePictureURL(ctx context.Context, dev *t
 }
 
 func (is *IdentityServer) updateEndDevice(ctx context.Context, req *ttnpb.UpdateEndDeviceRequest) (dev *ttnpb.EndDevice, err error) {
-	if err = rights.RequireApplication(ctx, req.EndDeviceIdentifiers.ApplicationIdentifiers, ttnpb.RIGHT_APPLICATION_DEVICES_WRITE); err != nil {
+	if clusterauth.Authorized(ctx) == nil {
```

Comment:
Another solution would be to allow the cluster authentication to have the `RIGHT_APPLICATION_DEVICES_WRITE` right. I think that's too wide for what we are trying to solve here.

Reply:
Indeed, we could consider that for a future refactoring